### ISSUE TITLE
Probabilistic 3D Human Shape and Pose Estimation from Multiple Unconstrained Images in the Wild

### ISSUE THREAD

**Issue by Arata2887** [Arata2887/issue_copy#30]
### 0.どんな論文? /What does this work do?

- https://arxiv.org/pdf/2103.10978v2
- CVPR 2021
- Introduces **a novel task** of **estimating 3D human body shape and pose** from **a group of unconstrained images**. 
- Proposed a method to predict distributions over SMPL body shape and pose parameters and probabilistically combines these distributions to improve shape estimation accuracy.
- A synthetic evaluation dataset is created(unpublished).

### 1. 論文のモチベーション / Motivation(s) of the paper / document

> <img width="424" alt="Fig 1" src="https://github.com/Arata2887/issue_copy/assets/119403000/21cb5832-4aeb-40e2-a2d6-df38db7bb452">

- Current 3D human body shape and pose estimation methods mainly focus on single images, videos, or multi-view images, all of which have constraints on pose, camera viewpoint, or background.
- There is a need for a method that leverages the additional information present in multiple images of the same subject, even if those images are unconstrained.
- The **proposed method(Probabilistic Shape combination)** addresses this gap by utilizing multiple images to improve the accuracy of shape prediction and quantifies pose prediction uncertainty in challenging conditions.

### 2. 論文のキーアイデア / Key idea(s) of the paper / document

**Probabilistic Shape combination**
Utilizes proxy representations like silhouettes and 2D joint heatmaps to predict distributions over SMPL parameters, which are then combined probabilistically.

> <img width="775" alt="Fig 2" src="https://github.com/Arata2887/issue_copy/assets/119403000/0d357f59-4030-42ef-a5ae-aeb6a33ed20d">
> 
> **Proxy Representation Computation**
> - Uses DensePose for body part segmentations converted to silhouettes and Keypoint-RCNN for 2D joint locations converted to heatmaps.
> - Combines these representations to form each input proxy representation.
> 
> **Distribution Prediction Network**
> - Predicts multivariate Gaussian distributions over SMPL pose and shape parameters for each image.
> - Combines the individual shape distributions from each image probabilistically to obtain a final body shape estimate.
> 
> **Training with Synthetic Data**
> - Uses synthetic training data to overcome the lack of diverse and accurate real-world datasets.
> - Augments synthetic data to model occlusions and missing body parts(like STRAPS), bridging the gap between synthetic and real inputs. 

This figure illustrates the robustness of the proposed method against clean and corrupted synthetic inputs, showing how the method handles occlusions and noise by quantifying prediction uncertainty.

**synthetic evaluation dataset**
A synthetic evaluation dataset is created for analysis, consisting of 1000 synthetic humans with randomly sampled SMPL body shapes. Each human is posed in 4 different SMPL poses from Human3.6M subjects 9 and 11, with the camera facing front, back, left, or right. For each human, there are 4 clean and 4 corrupted synthetic inputs.

> <img width="325" alt="Fig 3" src="https://github.com/Arata2887/issue_copy/assets/119403000/74dab8cb-fe3a-42b8-8d4c-487a94ef2078">

This figure shows the method's robustness against clean and corrupted inputs, handling occlusions and noise by quantifying prediction uncertainty.
- **Clean Synthetic Input:** Front, left, back, and right views of clean synthetic images.
- **Pose and Shape Distribution:** Predicted 3D pose and shape distributions for clean inputs, with uncertainty visualized (0cm to >20cm).
- **Shape Distribution:** Individual shape distribution predictions for clean inputs, with uncertainty visualized (0cm to >5cm).
- **Corrupted Synthetic Input:** Front, left, back, and right views of corrupted synthetic images with occlusions and noise.
- **Pose and Shape Distribution (Corrupted):** Predicted 3D pose and shape distributions for corrupted inputs, showing increased uncertainty.
- **Shape Distribution (Corrupted):** Individual shape distribution predictions for corrupted inputs, with visualized uncertainty.

### 3. その他（実験結果、関連研究、分からなかったところ 等）/ Misc (Experimental results, Related papers, Questions)

**Quantitative results:** 
**Table 3** Compares PVE-T-SC on SSP-3D, showing the proposed method's performance relative to other methods.

> <img width="385" alt="Table 3" src="https://github.com/Arata2887/issue_copy/assets/119403000/431942a4-fe17-45c8-bf00-ee45f22e7676">

**Table 4** Compares MPJPE-SC and MPJPE-PA on 3DPW, evaluating the accuracy of pose estimation.

> <img width="340" alt="Table 4" src="https://github.com/Arata2887/issue_copy/assets/119403000/6330b3e2-0b3a-4f48-921a-ede4567de000">

**Figure 6** Shows sorted distributions of PVE-T-SC on SSP-3D and MPJPE-PA on 3DPW, illustrating the method's performance compared to others.

> <img width="540" alt="Fig 6" src="https://github.com/Arata2887/issue_copy/assets/119403000/ada396c5-ecbd-487b-85d2-b9f4e28dfa16">

**Qualitative results:**  Figure 5 shows example predictions on groups of images from the SSP-3D dataset. It compares single-image predictions from SPIN, STRAPS, and the **proposed method**, along with the probabilistically combined body shapes from the proposed method.

> <img width="540" alt="Fig 6" src="https://github.com/Arata2887/issue_copy/assets/119403000/4a44557f-7dff-43ea-ac5f-1aa1e8fd5c49">

**Ablation studies:** In the **5.1 Ablation Studies** section, the authors conducted several ablation studies to evaluate model performance and uncertainty prediction. Since the results are presented in several different tables and figures, please refer to the paper for details.

> <img width="425" alt="Table 1" src="https://github.com/Arata2887/issue_copy/assets/119403000/bd7e4e14-26cc-4b40-b2fd-61d91253ec8e">

> <img width="419" alt="Table 2" src="https://github.com/Arata2887/issue_copy/assets/119403000/d4a2c3d9-8cf1-4832-b372-fa0a8638a982">

### 4.公開コードやデータセットは? /Public codes or datasets?

Not available
